apps/frontend/src/app/tournament-admin/tournament-form/tournament-form.component.html

```diff
@@ -0,0 +1,35 @@
+<form [formGroup]="myForm" (submit)="onSubmit()">
+  <div class="editTournamentData textAlign form-group">
+    <input matInput formControlName="name" id="editTournamentName" class="nameOfTournament" />
```

Comment:
Every input in a from needs to have a label associated with it if you want to have proper accessibility. If you have a label element where you are absolutely sure that it is not needed by sighted users, you can hide it using an sr-only class like this: https://stackoverflow.com/a/19758620
Add it to your styles.scss to use it.